lib/component.js

```diff
@@ -41,6 +41,9 @@ export default class Component extends WebComponent {
       this.isPanelChild = true;
       // find $panelParent
       for (let el = this.parentElement; el && !this.$panelParent; el = el.parentElement) {
+        if (el.nodeType === 11) {  // 11 = DOCUMENT_FRAGMENT_NODE
```

Comment:
what do you think about pulling this out into an actual `const DOCUMENT_FRAGMENT_NODE = 11;` at the top of the module?